packages/gutenpride-template/src/src/save.js.mustache

```diff
@@ -0,0 +1,19 @@
+/**
+ * Retrieves the translation of text.
+ *
+ * @see https://developer.wordpress.org/block-editor/packages/packages-i18n/
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * The save function defines the way in which the different attributes should
+ * be combined into the final markup, which is then serialized by the block
+ * editor into `post_content`.
+ *
+ * @see https://developer.wordpress.org/block-editor/developers/block-api/block-edit-save/#save
+ *
+ * @return {WPElement} Element to render.
+ */
+export default function Save( { attributes, className } ) {
+	return <div className={ className }>{ attributes.message }</div>;
```

Comment:
There was a fix added for the original template, see:
https://github.com/WordPress/gutenberg/blob/master/packages/create-block/lib/templates/esnext/src/save.js.mustache

Reply:
I added a fix for this in save.js.mustache. In edit.js.mustache,  Should I wrap a `<div { ...useBlockProps() }>` around `<TextControl>`?

Reply:
Good question. In theorhy, it should work with Input as well, but I don't know that :)